cwms-data-api/src/main/java/cwms/cda/ApiServlet.java

```diff
@@ -186,9 +197,27 @@ public void destroy() {
 
     @Override
     public void init(ServletConfig config) throws ServletException {
-        logger.atInfo().log("Initializing CWMS Data API Version:  " + obtainFullVersion(config));
+        String fullVersion = obtainFullVersion(config);
+        logger.atInfo().log("Initializing %s Version: %s", new Object[]{APPLICATION_TITLE, fullVersion});
         metrics = (MetricRegistry)config.getServletContext()
                 .getAttribute(MetricsServlet.METRICS_REGISTRY);
+        OtlpConfig otlpConfig = key -> {
+            //From the docs: https://micrometer.io/docs/registry/otlp
+            //If you instead of returning null, bind it to a property source (e.g.: a simple Map can work),
+            //you can override the default configuration through properties.
+            return null;
+        };
+
+        meterRegistry = new OtlpMeterRegistry(otlpConfig, Clock.SYSTEM);
```

Comment:
That's amazing, something that actually takes a "clock." Like, that's one of those really good ideas that literally noone actually does.